packages/website/src/styles/main.scss

```diff
@@ -3,69 +3,11 @@
 @import '@coveord/plasma-style/scss/variables';
 @import 'syntax-highlighting';
 
-/* stylelint-disable-next-line selector-id-pattern */
```

Comment:
It doesn't fail if you run `pnpm lint` ?

Reply:
I added the comment back